message.go

```diff
@@ -30,11 +30,7 @@ func (mm *MessageManager) Listen(messageType string, handler MessageHandler) {
 	mm.listeners[messageType] = append(mm.listeners[messageType], handler)
 }
 
-// WindowResizeMessage is a message that's being dispatched whenever the game window is being resized by the gamer
-type WindowResizeMessage struct {
-	OldWidth, OldHeight int
-	NewWidth, NewHeight int
+// Remove removes all listeners of a message type
+func (mm *MessageManager) Remove(messageType string) {
```

Comment:
Yes that would be the next step. Maybe have instead a `RemoveType` and a `Remove`